plugins/clopen.py

```diff
@@ -530,6 +549,101 @@ async def on_submit(self, interaction: Interaction) -> None:
             return
         await interaction.response.send_message("\u2705", ephemeral=True, delete_after=60)
 
+def owner_only(func: Callable[..., Awaitable[None]]):
+    async def wrapper(self: "PromptView", interaction: Interaction, *args: Any, **kwargs: Any) -> None:
+        if interaction.user.id != self.owner:
+            await interaction.response.send_message("This isn't yours.", ephemeral=True,
+                delete_after=60)
+            return
+        await func(self, interaction, *args, **kwargs)
+    return wrapper
+
+class PromptView(View):
+    def __init__(self, channel: TextChannel, owner: int) -> None:
+        super().__init__(timeout=None)
```

Comment:
Sorry, this approach is fundamentally flawed.
You're creating views with no timeout, meaning the view objects are stored in discord.py and never disposed of, leaking memory.
A view should either have a timeout or be fully stateless/persistent.
For persistent views we currently simply assign them `custom_id`'s and then manually intercept their interactions with an `on_interaction`. The library has since added support for this use-case in the form of `DynamicItem`, c.f. https://github.com/Rapptz/discord.py/blob/master/examples/views/persistent.py , we should migrate to that sometime.
You could try and painstakingly call `.close()` on the views that are no longer in use, but that seems prone to error and  will not work across a bot restart/plugin reload.